pynxtools/dataconverter/writer.py

```diff
@@ -158,6 +159,11 @@ def handle_dicts_entries(data, grp, entry_name, output_path, path):
                                compression_opts=strength)
         else:
             grp.create_dataset(entry_name, data=data["compress"])
+    else:
+        print("comes here!!")
```

Comment:
@sherjeelshabih Is this intentional?

Reply:
Nope. I'll fix this in a separate PR now.